src/commonComponents/cart/Cart.js

```diff
@@ -0,0 +1,75 @@
+// prop-types
+import PropTypes from "prop-types";
+
+// lodash
+import _noop from "lodash/noop";
+
+//helpers
+import {
+  getCartWithProducts,
+  getTotalAmountOfCart,
+  getTotalItemsInCart,
+} from "./cart.helpers";
+
+// components
+import Button, { TYPES } from "../button";
+import CartItems from "./components/cartItems";
+
+// css
+import "./cart.css";
+import { useMemo } from "react";
+
+const Cart = (props) => {
+  const {
+    cart: { items: cartItems },
+    products,
+    onCartChange,
+    onCheckout,
+  } = props;
+
+  const itemsCount = getTotalItemsInCart(cartItems);
+
+  const cartWithProducts = useMemo(() => {
+    return getCartWithProducts(products, cartItems);
+  }, [products, cartItems]);
+
+  const totalAmount = getTotalAmountOfCart(cartWithProducts);
+
+  return (
+    <div className="flex-column justify-content-space-between">
+      <div className="cart-header">
+        <h3>Cart</h3>
+        <p className="sub-title">{itemsCount} items</p>
```

Comment:
You can use the getSubHeading helper from menusection

move that getSubHeading a up in the folder structure